cg/cli/get.py

```diff
@@ -0,0 +1,96 @@
+import re
+
+import click
+from tabulate import tabulate
+
+from cg.store import Store
+
+SAMPLE_HEADERS = ['Sample', 'Name', 'Customer', 'Application', 'State', 'Priority', 'External?']
+FAMILY_HEADERS = ['Family', 'Name', 'Customer', 'Priority', 'Action']
+FLOWCELL_HEADERS = ['Flowcell', 'Type', 'Sequencer', 'Date', 'Archived?']
+
+
+@click.group(invoke_without_command=True)
+@click.argument('identifier', required=False)
+@click.pass_context
+def get(context, identifier):
+    """Get information about records in the database."""
+    context.obj['status'] = Store(context.obj['database'])
+    if identifier and re.match(r'[A-Z]{3}[0-9]{4,5}[A-Z]{1}[1-9]{1,3}', identifier):
+        context.invoke(sample, sample_ids=[identifier])
+    elif identifier and re.match(r'^[a-z]*$', identifier):
+        # try family information
+        context.invoke(family, family_id=identifier)
+    elif identifier and re.match(r'[HC][A-Z0-9]{8}', identifier):
```

Comment:
no `^` and `$`?

Reply:
hmm didn't think of that but yes, that makes sense!